chrome/settings-navigation.json

```diff
@@ -80,9 +80,10 @@
             "expandable": true,
             "permissions": [
                 {
-                    "method": "isOrgAdmin",
-                    "apps": [
-                        "cost-management"
+                    "method": "isEntitled",
+                    "args": [
+                        "insights",
```

Comment:
@ryelo So the `isEentitled` takes only one parameter at one time, so this won't work. You can use multiple `isEntitled` methods, but that does `AND` so all permission must be `true` in order to show the item. So I guess we need `OR` evaluation then.